KeyManager.Library.KeyStore.SAM_SE.UI/SAM_SESymmetricKeyEntryPropertiesControl.xaml

```diff
@@ -0,0 +1,429 @@
+                                    MinValue="0" MaxValue="13" CurrentValue="{Binding SAM_SEProperties.DESFire.KeyNum, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                                    Hint="{x:Static properties:Resources.KeyEntryDESFireKeyNumber}"/>
+                            </Grid>
+                            <wpfappctrls:NumericUpDownControl Grid.Row="4" Margin="0,5,0,5" IsEnabled="{Binding SAM_SEProperties.DESFire.ParamEnable}"
+                                    MinValue="0" MaxValue="31" CurrentValue="{Binding SAM_SEProperties.DESFire.FileNum, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                                    Hint="{x:Static properties:Resources.KeyEntryDESFireFileNum}"/>
+                            <Grid Grid.Row="5" Margin="0,5,0,5" IsEnabled="{Binding SAM_SEProperties.DESFire.ParamEnable}">
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition />
+                                    <ColumnDefinition />
+                                </Grid.ColumnDefinitions>
+                                <wpfappctrls:NumericUpDownControl Grid.Column="0" Margin="0,0,5,0"
+                                     MinValue="1" MaxValue="8"
+                                     CurrentValue="{Binding SAM_SEProperties.DESFire.Size, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                                     Hint="{x:Static properties:Resources.KeyEntryDESFireSize}"/>
+                                <wpfappctrls:NumericUpDownControl Grid.Column="1" Margin="5,0,0,0"
+                                     MinValue="0" MaxValue="{Binding SAM_SEProperties.DESFire.OffsetMax, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                                     CurrentValue="{Binding SAM_SEProperties.DESFire.Offset, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                                     Hint="{x:Static properties:Resources.KeyEntryDESFireOffset}"/>
+                            </Grid>
+                            <Grid Grid.Row="6" Margin="0,5,0,5" IsEnabled="{Binding SAM_SEProperties.DESFire.ParamEnable}">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition/>
+                                    <RowDefinition/>
+                                    <RowDefinition/>
+                                    <RowDefinition/>
+                                </Grid.RowDefinitions>
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition/>
+                                    <ColumnDefinition/>
+                                </Grid.ColumnDefinitions>
+                                <TextBlock Grid.Row="0" Grid.Column="0" Margin="0,0,0,0"
+                                    Text="{x:Static properties:Resources.KeyEntryDESFireEncryption}"
+                                    FontSize="9" Foreground="DarkGray"/>
```

Comment:
Hardcoded colors should be avoided to be sure it works with Light/Dark themes. It would be better to reference MaterialDesign theme colors. This comment is for all hardcoded colors on this file.